src/components/workspace/ControlBar.tsx

```diff
@@ -279,22 +282,22 @@ class ControlBar extends React.PureComponent<ControlBarProps, { joinElemValue: s
     );
 
     return (
-      <div className="ControlBar_editor pt-button-group">
+      <div className={classNames('ControlBar_editor', Classes.BUTTON_GROUP)}>
         {toggleAutorunButton}
         {this.props.isEditorAutorun
           ? autoRunButton
           : this.props.isRunning
-            ? stopButton
-            : this.props.isDebugging
-              ? null
-              : runButton}
+          ? stopButton
+          : this.props.isDebugging
+          ? null
+          : runButton}
```

Comment:
Let's help refactor this since the prettier has so much trouble prettifying this.

```javascript
{
  {
    if (this.props.isEditorAutorun) {
      return autoRunButton;
    }
    if (this.props.isRunning) {
      return stopButton;
    }
    if (this.props.isDebugging) {
      return null;
    }
    return runButton;
  }
}
```